includes_php71/class-wcmp-export.php

```diff
@@ -0,0 +1,2100 @@
+            // standard = default, overwrite if options found
+            $delivery_type = 'standard';
+            if ( ! empty($myparcelbe_delivery_options)) {
+                // pickup & pickup express store the delivery type in the delivery options,
+                if (empty($myparcelbe_delivery_options['price_comment']) && ! empty($myparcelbe_delivery_options['time'])) {
+                    // check if we have a price_comment in the time option
+                    $delivery_time = array_shift($myparcelbe_delivery_options['time']); // take first element in time array
+                    if (isset($delivery_time['price_comment'])) {
+                        $delivery_type = $delivery_time['price_comment'];
+                    }
+                } else {
+                    $delivery_type = $myparcelbe_delivery_options['price_comment'];
+                }
+            }
+
+            // backwards compatibility for pakjegemak
+            if ($pgaddress = WCX_Order::get_meta($order, '_myparcelbe_pgaddress')) {
+                $delivery_type = 'retail';
+            }
+
+            // convert to int (default to 2 = standard for unknown types)
+            $delivery_type = isset($delivery_types[$delivery_type]) ? $delivery_types[$delivery_type] : 2;
+
+            return $delivery_type;
+        }
+        //options.delivery_date custom delivery date not supported for carrier bpost
+        //public function get_delivery_date($order, $myparcelbe_delivery_options = '') {
+        //    if (empty($myparcelbe_delivery_options)) {
+        //        $myparcelbe_delivery_options = WCX_Order::get_meta($order, '_myparcelbe_delivery_options');
+        //    }
+        //
+        //    if ( ! empty($myparcelbe_delivery_options) && ! empty($myparcelbe_delivery_options['date'])) {
+        //        $delivery_date = $myparcelbe_delivery_options['date'];
+        //
+        //        $delivery_type = $this->get_delivery_type($order, $myparcelbe_delivery_options);
+        //        if (in_array($delivery_type, array(1, 3)) && ! empty($myparcelbe_delivery_options['time'])) {
+        //            $delivery_time_options = array_shift(
+        //                $myparcelbe_delivery_options['time']
+        //            ); // take first element in time array
+        //            $delivery_time = $delivery_time_options['start'];
+        //        } else {
+        //            $delivery_time = '00:00:00';
+        //        }
+        //        $delivery_date = "{$delivery_date} {$delivery_time}";
+        //
+        //        return $delivery_date;
+        //    } else {
+        //        return false;
+        //    }
+        //}
+
+        public function get_order_shipping_class($order, $shipping_method_id = '')
+        {
+            if (empty($shipping_method_id)) {
+                $order_shipping_methods = $order->get_items('shipping');
+
+                if ( ! empty($order_shipping_methods)) {
+                    // we're tak
```

Comment:
de `get_tier_ranges` kan denk ik weg, is namelijk voor de Digitale postzegel

Reply:
done